spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/core/PubSubConfigurationTests.java

```diff
@@ -122,18 +135,22 @@ void testSubscriberFlowControlSettings() {
   @Test
   void testComputeFlowControlSettings_returnCustom() {
     PubSubConfiguration pubSubConfiguration = new PubSubConfiguration();
+
     PubSubConfiguration.Subscriber subscriber = new PubSubConfiguration.Subscriber();
     PubSubConfiguration.FlowControl selectiveFlowControl = subscriber.getFlowControl();
 
     selectiveFlowControl.setLimitExceededBehavior(FlowController.LimitExceededBehavior.Ignore);
     selectiveFlowControl.setMaxOutstandingElementCount(1L);
     selectiveFlowControl.setMaxOutstandingRequestBytes(2L);
 
-    pubSubConfiguration
-        .getSubscription()
-        .put("projects/projectId/subscriptions/subscription-name", subscriber);
+    ProjectSubscriptionName psn =
+        ProjectSubscriptionName.parse("projects/projectId/subscriptions/subscription-name");
```

Comment:
How about we put this string in a variable?

Reply:
extracting as global constant `QUALIFIED_SUBSCRIPTION_NAME`